packages/containers_common.rb

```diff
@@ -0,0 +1,119 @@
+# Adapted from Arch Linux containers-common PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/containers-common/trunk/PKGBUILD
+
+require 'package'
+
+class Containers_common < Package
+  description 'Configuration files and manpages for containers'
+  homepage 'https://github.com/containers'
+  version '0.49.3'
+  license 'Apache'
+  compatibility 'all'
+  source_url 'https://github.com/containers/common.git'
+  git_hashtag "v#{version}"
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containers_common/0.49.3_armv7l/containers_common-0.49.3-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containers_common/0.49.3_armv7l/containers_common-0.49.3-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containers_common/0.49.3_i686/containers_common-0.49.3-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containers_common/0.49.3_x86_64/containers_common-0.49.3-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '06f20054310a55b9371336b4b5cd88b6a9f89d3f85702d1e99b21aec7fa13123',
+     armv7l: '06f20054310a55b9371336b4b5cd88b6a9f89d3f85702d1e99b21aec7fa13123',
+       i686: '80811ec2080d153dd4f470c7e92413dc8cc61526241749391237a2eec86e0405',
+     x86_64: '1597d590420e90de5edbca768026642871c0849aeb7566b8ed55a22be7dc9d99'
+  })
+
+  depends_on 'netavark'
+  depends_on 'go_md2man' => :build
+
+  def self.build
+    @image_version = 'v5.20.0'
+    @podman_version = 'v4.0.2'
+    @shortnames_version = 'v2022.02.08'
+    @skopeo_version = 'v1.6.1'
+    @storage_version = 'v1.38.2'
```

Comment:
Should these be separate packages?

Reply:
no, I don't think we need these other packages...